backend/engine/processor/sbom_cdx.py

```diff
@@ -35,15 +35,26 @@ def process_dependency(dep: dict, scan: Scan) -> None:
     licenses = []
     for license in dep["licenses"]:
         license_id = license.get("id").lower()
+        # Character limit for licenses are 256 and Trivy SBOM at times can incorrectly
+        # return out lengthy gibberish
+        if len(license.get("name")) > 256:
+            logger.error(f"{component}'s license exceeds character limit: {license['name']}")
```

Comment:
Nit: Phrasing of the error could imply that the character limit is whatever come sout of `license['name']`